Add documentation to Charmhub (#23)
canonical/self-signed-certificates-operator: docs/index.md

The Self Signed Certificates Operator provides self-signed X.509 certificates to your charms.

This charm relies on the `tls-certificates` charm relation interface. When a requirer charm inserts a Certificate Signing Request in its unit databag, the `self-signed-certificates-operator` will read it, generate a self-signed X.509 certificates and
inserts this certificate back into the relation data.

> This charm [Self-Signed-Certificates-Operator](https://github.com/canonical/self-signed-certificates-operator) is useful when developing charms or when deploying charms in non-production environment on top of [Juju](https://juju.is/).

Reviewer: ```suggestion
The [Self-Signed-Certificates-Operator](https://github.com/canonical/self-signed-certificates-operator) is useful when developing charms or when deploying charms in non-production environment on top of [Juju](https://juju.is/).
```

Author: OK

> The Self Signed Certificates Operator provides self-signed X.509 certificates to your charms.

Reviewer: Ultimately I think most of our docs for this simple charm can be in this index. It's a fairly simple charm at the end of the day and we probably don't want to maintain 5-10 pages of text for it.
